src/odata-to-abstract-sql.ts

```diff
@@ -991,11 +1004,16 @@ export class OData2AbstractSQL {
 		fieldName: string,
 		computed?: AbstractSqlQuery,
 		alias: string = fieldName,
+		alwaysCompileComputed: boolean = false,
```

Comment:
Can you add a comment giving a brief/basic explanation of what the flag is for so that future people looking at the code have a starting point for understanding it please

Reply:
Added a comment and found a better flag name